src/Uno.UI.Composition/Composition/Visual.skia.cs

```diff
@@ -123,6 +123,33 @@ internal Matrix4x4 TotalMatrix
 			}
 
 			return _totalMatrix;
+
+			Matrix4x4 GetTransform()
```

Comment:
Should we have a `CombinedMatrix` also here ?

Reply:
Right now, probably not. I added a new `CombinedTransformMatrix` property in CompositionShape because I needed to listen to changes to it. If we later need something similar here (e.g. so that we know exactly when to recalculate the TotalMatrix, instead of the current naive MatrixDirty logic), we can go ahead and introduce a similar CombinedMatrix property. Maybe in another PR.